src/test/java/org/eclipse/emfcloud/jackson/TestSuite.java

```diff
@@ -12,6 +12,7 @@
 
 import org.eclipse.emfcloud.jackson.databind.type.EcoreTypeFactoryTest;
 import org.eclipse.emfcloud.jackson.tests.AnnotationTest;
+import org.eclipse.emfcloud.jackson.tests.ArrayTest;
```

Comment:
Could you please update the copyright header with the latest year:
`Copyright (c) 2019-2022 Guillaume Hillairet and others.`

Reply:
Done